scripts/cmake/vcpkg_configure_make.cmake

```
@@ -512,6 +512,17 @@ function(vcpkg_configure_make)
         list(APPEND _csc_OPTIONS --disable-shared --enable-static)
     endif()
 
+    # Can be set in the triplet to append options for configure
+    if(DEFINED VCPKG_MAKE_CONFIGURE_OPTIONS)
+        list(APPEND _csc_OPTIONS "${VCPKG_MAKE_CONFIGURE_OPTIONS}")

```

```suggestion
        list(APPEND _csc_OPTIONS ${VCPKG_MAKE_CONFIGURE_OPTIONS})
```

Since VCPKG_MAKE_CONFIGURE_OPTIONS is a list variable in this context, I believe it should be expanded without quotes @strega-nil @strega-nil-ms

---

Applied 👍